beta/docker/README.md

```diff
@@ -0,0 +1,231 @@
+
+1. Copy the Google Workspace service account key to the working directory.
+
+   > **Note**
+   >
+   > If you are deploying SCIM bridge directly on the Linux server, you will need to transfer the Workspace credentials file
+   > from your local machine that is signed in to Google (or your 1Password account if you saved the credentials there), for example:
+   >
+   > ```sh
+   >  scp ./workspace-credentials.json user@host:scim-examples/beta/docker/workspace-credentials.json
+   > ```
+
+2. Copy the [settings file template](/beta/workspace-settings.json) from this repository:
+
+    ```sh
+    cp ../workspace-settings.json ./workspace-settings.json
+    ```
+
+    Edit the file. Replace the values for each key:
+
+    - `actor`: the email address for the administrator that the service account is acting on behalf of
+    - `bridgeAddress`: the URL for your SCIM bridge based on the DNS record created in [Get started](#get-started)
+
+    ```json
+    {
+        "actor":"admin@example.com",
+        "bridgeAddress":"https://scim.example.com"
+    }
+    ```
+
+    Save the file.
+
+## Deploy 1Password SCIM bridge
+
+After the DNS record has propagated, set the domain name for your SCIM bridge inline, use the template to output a canonical configuration that will create a Docker secret and deploy SCIM bridge, then create the stack from this configuration. You SCIM bridge should automatically acquire and manage a TLS certificate from Let's Encrypt on your behlaf using the supplied domain name.
+
+*Example command:*
+
+```sh
+OP_TLS_DOMAIN=scim.example.com docker stack config --compose-file ./compose.template.yaml |
+    docker stack deploy --compose-file - op-scim-bridge
+```
+
+Replace `scim.example.com` with the fully qualified domain name that points to your Linux server, then run the command to deploy your SCIM bridge.
+
+### Connect your SCIM bridge to Google Workspace
+
+If you are integrating with Google Workspace, use `compose.gw.yaml` to create the additional Docker secrets needed for Workspace.
+
+*Example command:*
+
+```sh
+OP_TLS_DOMAIN=scim.example.com docker stack config \
+    --compose-file ./compose.template.yaml \
+    --compose-file ./compose.gw.yaml |
+        docker stack deploy --compose-file - op-scim-bridge
+```
+
+Replace `scim.example.com` with the fully qualified domain name that points to your Linux server, then run the command to deploy SCIM bridge with access to the required configuration and credentials.
+
+## Test your SCIM bridge
+
+Run this command to retrieve logs from the service for the SCIM bridge container:
+
+```sh
+docker service logs op-scim-bridge_scim
+```
+
+You can sign in to the URL of your SCIM bridge based on its fully qualified domain name (for example, <https://scim.example.com/>). Sign in with your bearer token to test the connection, view status information, or retrieve logs.
```

Comment:
We should be consistent throughout on how we reference the FQDN or the DNS record in our example wording on what to replace.